modules/FindBasisUniversal.cmake

```diff
@@ -115,100 +73,162 @@ macro(_basis_setup_source_file source)
     endif()
 endmacro()
 
+include(FindPackageHandleStandardArgs)
 # Find components
 foreach(_component ${BasisUniversal_FIND_COMPONENTS})
     if(_component STREQUAL "Encoder")
         if(NOT TARGET BasisUniversal::Encoder)
-            find_path(BasisUniversalEncoder_DIR NAMES basisu_frontend.cpp
-                HINTS "${BASIS_UNIVERSAL_DIR}" "${BASIS_UNIVERSAL_DIR}/encoder"
-                NO_CMAKE_FIND_ROOT_PATH)
-            if(BasisUniversalEncoder_DIR)
-                set(BasisUniversal_Encoder_FOUND TRUE)
+            # Try to find compiled library, e.g. when installed through vcpkg
+            find_library(BasisUniversalEncoder_LIBRARY_RELEASE basisu_encoder)
+            find_library(BasisUniversalEncoder_LIBRARY_DEBUG basisu_encoder HINTS "debug")
+
+            if(BasisUniversalEncoder_LIBRARY_DEBUG AND BasisUniversalEncoder_LIBRARY_RELEASE)
+                find_path(BasisUniversalEncoder_INCLUDE_DIR basisu_enc.h PATH_SUFFIXES "basisu/encoder")
+
+                add_library(BasisUniversal::Encoder UNKNOWN IMPORTED)
+                set_property(TARGET BasisUniversal::Encoder APPEND PROPERTY
+                    IMPORTED_LOCATION_DEBUG ${BasisUniversalEncoder_LIBRARY_DEBUG})
+                set_property(TARGET BasisUniversal::Encoder APPEND PROPERTY
+                    IMPORTED_LOCATION_RELEASE ${BasisUniversalEncoder_LIBRARY_RELEASE})
+                set_property(TARGET BasisUniversal::Encoder APPEND PROPERTY
+                    INTERFACE_INCLUDE_DIRECTORIES ${BasisUniversalTranscoder_INCLUDE_DIR})
+
+                find_package_handle_standard_args(BasisUniversal_Encoder
+                    BasisUniversalEncoder_LIBRARY_DEBUG
+                    BasisUniversalEncoder_LIBRARY_RELEASE
+                    BasisUniversalEncoder_INCLUDE_DIR)
```

Comment:
Calling this here would mean the `-- Found BasisUniversal` message gets printed multiple times. This should be setting the `BasisUniversal_Encoder_FOUND` variable instead if all the relevant variables get found, like it does in the `else()` branch. For example,

```cmake
if((BasisUniversalEncoder_LIBRARY_DEBUG OR BasisUniversalEncoder_LIBRARY_RELEASE) AND BasisUniversalEncoder_INCLUDE_DIR)
    set(BasisUniversal_Encoder_FOUND TRUE)
endif()
```

Same for the transcoder below.